src/mongo/tools/SConscript

```diff
@@ -43,6 +43,7 @@ perconadecrypt = env.Program(
     LIBDEPS=[
         '$BUILD_DIR/mongo/db/encryption/encryption_options',
         '$BUILD_DIR/mongo/db/encryption/encryption_vault',
+        '$BUILD_DIR/mongo/db/encryption/encryption_kmip',
```

Comment:
This does not compile because there is no 'encryption_kmip' library defined in src/mongo/db/encryption/SConscript.
Let's extract encryption_kmip from encryption_vault

Reply:
Fixed.